src/Microsoft.CodeAnalysis.Analyzers/UnitTests/FixAnalyzers/FixerWithFixAllAnalyzerTests.cs

```diff
@@ -250,11 +251,11 @@ public override Task RegisterCodeFixesAsync(CodeFixContext context)
             // Test0.cs(12,7): warning RS1016: 'C1' registers one or more code fixes, but does not override the method 'CodeFixProvider.GetFixAllProvider'. Override this method and provide a non-null FixAllProvider for FixAll support, potentially 'WellKnownFixAllProviders.BatchFixer', or 'null' to explicitly disable FixAll support.
             var missingGetFixAllProviderOverrideDiagnostic = GetCSharpOverrideGetFixAllProviderExpectedDiagnostic(12, 7, "C1");
 
-            TestCSharpCore(source, missingGetFixAllProviderOverrideDiagnostic, validationMode: TestValidationMode.AllowCompileErrors);
+            await TestCSharpCore(source, missingGetFixAllProviderOverrideDiagnostic, compilerDiagnostics: CompilerDiagnostics.None);
```

Comment:
When you say new test framework, are you referring to the one we are trying to put in place through this PR or a new version coming up? Cause I am fine doing another commit adding those new diagnostics.

Reply:
> When you say new test framework, are you referring to the one we are trying to put in place through this PR or a new version coming up?

I mean the one you are using in this pull request.

> Cause I am fine doing another commit adding those new diagnostics.

Let's make it a separate PR, since we'll need to carefully review those and it could take more time.

Reply:
See https://github.com/dotnet/roslyn-analyzers/issues/2995